### ISSUE TITLE
Screen blink

### ISSUE THREAD

**Issue by plehegar** [w3c/webex#6]
WebEx on desktop notifies of many changes of state (every caller joining and leaving, chat activity, etc.) by blinking the application window and icon incessantly. This cannot be turned off via WebEx or operating system settings. The distraction is most certainly disabling for many users. This is also a failure of WCAG 2.0 and W3C site policy. Short of a fix from Cisco or a switch to a better telephony solution, the most optimal fix is to minimize or occlude the window with other windows, and cover the icon with tape on the monitor. Really.
